UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -383,45 +381,54 @@ public bool IsMentor(string userID)
 			if (response == null) return false;
 		}
 
-		//allow error response for local offline testing
+		//Must have non-null/empty username
+		if (string.IsNullOrEmpty(unverifiedConnPlayer.Username))
```

Comment:
oh yeah, that reminds me we have no way of validating Username, that needs to be added at some point